indigo/plugins.py

```diff
@@ -30,7 +30,7 @@ def for_locale(self, topic, country=None, language=None, locality=None):
         """ Find an appropriate importer for this locale description. Tightest match wins.
         """
         target = (country, language, locality)
-        match = self.lookup(topic, target, self.registry[topic].itervalues())
+        match = self.lookup(topic, target, iter(self.registry[topic].values()))
```

Comment:
`.values()` is already an iterable, doesn't need `iter()`
```suggestion
        match = self.lookup(topic, target, self.registry[topic].values())
```